backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5573610_sys_gh6310_UpdateSysConfigForHUEditorViewFactoryLayout.sql

```diff
@@ -0,0 +1 @@
+update ad_sysconfig set value = 'N' where name = 'de.metas.ui.web.handlingunits.HUEditorViewFactory.AlwaysUseSameLayout';
```

Comment:
i'm not 100% sure what this `AD_SysConfig` does..
it's used in `HUEditorViewFactoryTemplate`, which has five other subclasses, besides `ServiceHUEditorViewFactory`...

:question: ...so, in other words: are we sure this doesn't meddle in bad ways with other HU-Editors?

Also, please consider adding a description to this AD_SysConfig (->boyscout rule)

Reply:
@metas-ts I removed the script it was not needed

> i'm not 100% sure what this `AD_SysConfig` does..
> it's used in `HUEditorViewFactoryTemplate`, which has five other subclasses, besides `ServiceHUEditorViewFactory`...
> 

 I removed the script it was not needed